tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -617,6 +617,9 @@ class MaxSlotHelper : public ROOT::Detail::RDF::RActionImpl<MaxSlotHelper> {
    void Exec(unsigned int slot, unsigned int /*slot2*/) { fMaxSlots[slot] = std::max(fMaxSlots[slot], slot); }
    void Finalize() { *fMaxSlot = *std::max_element(fMaxSlots.begin(), fMaxSlots.end()); }
 
+   // Helper for RMergeableValue
+   void* GetMergeableValue() { return nullptr; }
```

Comment:
having to add this means this patch is backward-incompatible :D

Reply:
:sweat_smile: